frontend/src/pages/pipelines/GlobalModelCustomizationRoutes.tsx

```diff
@@ -10,7 +10,7 @@ const GlobalModelCustomizationRoutes: React.FC = () => (
   <ProjectsRoutes>
     <Route path="/" element={<GlobalModelCustomization />} />
     <Route
-      path="instructlab/:namespace?/*"
+      path="fine-tune/:modelRegistry/:registeredModelId/:modelVersionId/:namespace"
```

Comment:
I thought this would be in the routes for the model registry. Something like `/modelRegistry/:modelRegistry/registeredModels/:registeredModelId/versions/:modelVersionId/fine-tune/:namespace`, did I understand it wrongly? Because I saw the [mockups](https://www.figma.com/design/97akqulXiGIGtvxkdPLjMm/Data-connections-enhancement?node-id=5205-9129&m=dev) are under model registry context.

Reply:
i looked deeper into it and 

- model registry has a lot of mr specific ux that we dont want in our form. it would be a big refactor
- Andrew mentioned that we dont want to be merging this too much with mr. MR will pass it off to us in pipelines realm but we dont live under MR